convert2rhel/checks.py

```diff
@@ -136,22 +137,58 @@ def check_convert2rhel_latest():
         )
         return
 
-    convert2rhel_versions = re.findall(PKG_NEVR, raw_output_convert2rhel_versions, re.MULTILINE)
-    logger.debug("Found %s convert2rhel package(s)" % len(convert2rhel_versions))
+    # convert the raw output of convert2rhel version strings into a list
+    raw_output_convert2rhel_versions = raw_output_convert2rhel_versions.splitlines()
+
+    temp_raw_output = []
+
+    # We are expecting an repoquery output to be similar to this:
+    # C2R convert2rhel-0:0.17-1.el7.noarch
+    # We need the `C2R` identifier to be present on the line so we can know for
+    # sure that the line we are working with is the a line that contains
+    # relevant repoquery information to our check, otherwise, we just log the
+    # information as debug and do nothing with it.
+    for raw_version in raw_output_convert2rhel_versions:
+        if "C2R" in raw_version:
+            temp_raw_output.append(raw_version.lstrip("C2R "))
+        else:
+            # Mainly for debugging purposes to see what is happening if we got
+            # anything else that does not have the C2R identifier at the start
+            # of the line.
+            logger.debug("Got a line without the C2R identifier: %s" % raw_version)
```

Comment:
Same with this, we should not have tests that assert this @pr-watson. If there are any added I didn't see them, but same applies there

Reply:
I'll remove this as well